packages/core/src/utils/createMap/makeShadowRoot.ts

```diff
@@ -0,0 +1,80 @@
+const waitTime = 500
+const maxAttempts = 10
+
+/*
+ * CSS code taken from vuetify@2. While most of the vuetify@2 CSS is pulled to
+ * within the shadow, this is required on the outside to make it all work.
+ * Comments are from CSS source.
+ */
+const makeContainerCss = (containerId: string) => `
+#${containerId} {
+  box-sizing: border-box;
+  /* All browsers without overlaying scrollbars */
+  -webkit-text-size-adjust: 100%;
+  /* Prevent adjustments of font size after orientation changes in iOS */
+  word-break: normal;
+  -moz-tab-size: 4;
+  tab-size: 4;
+}
+
+#${containerId} * {
+  background-repeat: no-repeat;
+  /* Set \`background-repeat: no-repeat\` to all elements and pseudo elements */
+  box-sizing: inherit;
+}
+`
+
+const buildShadowDom = (
+  shadowHost: HTMLElement,
+  containerId: string
+): ShadowRoot => {
+  const wrapper = document.createElement('div')
+  wrapper.id = `${containerId}-wrapper`
+  const style = document.createElement('style')
+  style.innerHTML = makeContainerCss(containerId)
+
+  shadowHost.parentElement?.insertBefore(wrapper, shadowHost)
+  wrapper.appendChild(shadowHost)
+  wrapper.appendChild(style)
```

Comment:
No, this additional wrapper sets required CSS so that the map properly renders. Previously, all vuetify CSS was in the Root/Light DOM, and with it, some lines required to make the client work. I've identified those lines and added them progammatically. An alternative would've been to instruct users via API.md which CSS properties are needed in the wrapper, but that felt like delegating a burden that can be solved in a few lines automagically.